src/schema/users_model.py

```diff
@@ -0,0 +1,38 @@
+from datetime import datetime, timezone
+
+from pydantic import BaseModel, EmailStr, Field
+
+from src.schema.questions_model import Level
+
+
+class User(BaseModel):
+    """
+    Model to hold  user data.
+    """
+
+    name: str = Field(min_length=10)
+    password: str = Field(min_length=7)
+    email: EmailStr
+    linked_in: str = Field(min_length=8)
+    skill: list[str]
```

Comment:
if were having a list of strings, this list would be populated from the FE? we'll need an update route to add more skills I think. how about if skills is a standalone model, that can be updated by itself?